nixos/modules/services/web-apps/mediawiki.nix

```diff
@@ -8,7 +8,7 @@ let
   cfg = config.services.mediawiki;
   fpm = config.services.phpfpm.pools.mediawiki;
   user = "mediawiki";
-  group = if cfg.webserver == "apache" then "apache" else "mediawiki";
+  group = if cfg.webserver == "apache" then config.services.httpd.group else "mediawiki";
```

Comment:
This is now the same as it was before https://github.com/NixOS/nixpkgs/pull/227990